flink-ai-flow/ai_flow/protobuf/proto/scheduling_service.proto

```diff
@@ -39,7 +40,14 @@ message WorkflowExecutionRequest {
     string namespace = 1;
     string execution_id = 2;
     string workflow_name = 3;
-    map<string, string> args = 4;
+    google.protobuf.StringValue context = 4;
```

Comment:
The reason for using `google.protobuf.StringValue` instead of `string` is the context can be None?

Reply:
Yes